cmd/server/main.go

```diff
@@ -97,4 +101,9 @@ func main() {
 	if err := http.ListenAndServe(port, m); err != nil {
 		log.Println(err)
 	}
+	token,err := handlers.GeneratedGitJwtToken(ClientId,GitSecretKey)
```

Comment:
this line will never run, the call will be blocked on running http server. 
try running the code to see.

so I suggest to remove this lines, we will use it in another place

Reply:
that I didn't get that is why I place it here. should I write before the running the server for testing ?.

Reply:
nowhere.
we will use it later on fetching repository from github. so you can just remove this code.

btw you mixed client ID and github secret key, the arguments order is the opposite